packages/guilded.js/lib/managers/global/CalendarManager.ts

```diff
@@ -40,6 +50,9 @@ export class GlobalCalendarManager extends Base<number, CalendarEvent> {
     /** Delete a calendar event. */
     delete(channelId: string, calendarEventId: number): Promise<void> {
         return this.client.rest.router.deleteCalendarEvent(channelId, calendarEventId).then((data) => {
+            /* Delete from Cache???
+            this.client.calendars.cache.delete(calendarEventId);
```

Comment:
We can have an option in the client constructor for if the user wants to delete calendar events from cache. Take a look at other delete methods in the lib for inspiration